addons/mass_mailing/models/res_users.py

```diff
@@ -19,3 +19,10 @@ def systray_get_activities(self):
                 activity['name'] = _('Email Marketing')
                 break
         return activities
+
+    def write(self, values):
+        if values.get('email') and self == self.env.ref('base.user_admin'):
```

Comment:
Maybe `self.env.ref('base.user_admin') in self ` would be a bit more robust since it will handle cases where multiple records are edited and yours is one of them.

(If we can even talk about """robust""" for this feature...)

Reply:
Normally you will not write the same email on several users at once. And if you do, well ... probably you will have other issues than default contact data email value :D .